src/backend/distributed/commands/foreign_constraint.c

```diff
@@ -216,10 +216,11 @@ ErrorIfUnsupportedForeignConstraintExists(Relation relation, char referencingDis
 			ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 							errmsg("cannot create foreign key constraint "
 								   "since foreign keys from reference tables "
-								   "to distributed tables are not supported"),
-							errdetail("A reference table can only have foreign "
-									  "keys to other reference tables or citus "
-									  "local tables")));
+								   "and citus local tables to distributed tables "
```

Comment:
minor: kind of wondering whether citus local should just be "local"

Reply:
In general, I'm thinking about if we should do `s/citus local/local` in error messages, let's discuss this after this pr and let me open a seperate pr at the end of #4415 (if it makes sense to do this) ?